src/index.ts

```diff
@@ -222,6 +222,8 @@ class Spanner extends GrpcService {
   resourceHeader_: {[k: string]: string};
   routeToLeaderEnabled = true;
   directedReadOptions: google.spanner.v1.IDirectedReadOptions | null;
+  _instance_admin_api: null | undefined;
+  _database_admin_api: null | undefined;
```

Comment:
These fields are probably not needed, see my comment further down.

Reply:
Removed